dht/handler.go

```diff
@@ -99,3 +99,20 @@ func (dht *DHT) pongRequestHandler() {
 		}
 	}
 }
+
+func (dht *DHT) republishRequestHandler() {
+	for {
+		value := <- dht.db.LocalItemCh()
+
+		go func(val string) {
```

Comment:
I don't think we want to do this in a go routine, as we want to be somewhat near the alpha-parameter?

Reply:
Ok